packages/wonder-blocks-dropdown/utils/types.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+// TODO(sophie): incorporate this stuff later
+// export type ActionItemProps = {
+//     type: "action",
+//     /** Whether this item is disabled. Default false. */
+//     disabled?: boolean,
+//     /** Display text of the item. */
+//     label: string,
+//     /** URL to navigate to. */
+//     href?: string,
+//     /** Whether to use client-side navigation. */
+//     clientNav?: boolean,
+//     /** Callback on the action. */
+//     onClick?: () => void,
+// };
+
+export type SelectItemProps = {
+    type: "select",
+    /** Whether this item is disabled. Default false. */
```

Comment:
super nit: `/* ... */`

Reply:
Some other `type` files in the codebase use `/** ... */`. I will leave it like that to be consistent.

Reply:
I didn't realize that.